pupil_src/capture/run_capture.bat

```diff
@@ -1,2 +1,2 @@
-PATH %PATH%..\..\pupil_external
+SET PATH=%PATH%;%~dp0..\..\pupil_external
```

Comment:
Hi @PolynomialDivision - thanks for the updates. You are correct in your usage of `set`. I was just re-reading documentation and `setx` is the command want to avoid. `set` is safe to use as you note. 

`SET` 
>Display, set, or remove CMD environment variables. Changes made with SET will remain **only for the duration of the current CMD session**.

`SETX` 
> Set environment variables permanently, SETX can be used to set Environment Variables for the machine or currently logged on user.

I will test this again and can merge after testing or follow up with more comments.